zgw/openklant/src/main/kotlin/nl/nlportal/openklant/client/domain/Partijen.kt

```diff
@@ -191,4 +191,11 @@ fun CommonGroundAuthentication.asSoortPartij(): String =
         is BurgerAuthentication -> "persoon"
         is BedrijfAuthentication -> "organisatie"
         else -> "contactpersoon"
+    }
+
+fun CommonGroundAuthentication.asSoortPartijEnum(): SoortPartij =
+    when (this) {
+        is BurgerAuthentication -> SoortPartij.PERSOON
+        is BedrijfAuthentication -> SoortPartij.ORGANISATIE
+        else -> SoortPartij.CONTACTPERSOON
```

Comment:
This bothers me, but I don't think we have a mechanism of detecting a type of user for KeycloakAuthentication.